projects/docs-tools/src/modules/demo-page/demo-page.component.html

```diff
@@ -12,13 +12,17 @@
 
     <div class="sky-docs-demo-page-main">
       <sky-tabset *ngIf="enableTabLayout; else content" permalinkId="docs">
-        <sky-tab tabHeading="Design">
+        <sky-tab tabHeading="Design" *ngIf="!!designGuidelinesComponent">
```

Comment:
What's this?

Reply:
`enableTabLayout` now is true for if there is a design guidelines component OR if there are code examples. This `!!designGuidelinesComponent` used to be what `enableTabLayout` was set to, but because the tabs can be there with just "Development' and "Code examples" we now need this `ngIf` to hide the design guidelines if they don't exist.

Reply:
An example is `repeater`. If we don't add the `ngIf` here it will have an empty "Design" tab as we have tabs enabled due to having two tabs, but without the `ngIf` it will also render "Design" as an empty tab.